### DISCUSSION TITLE
FetS-AI on cuda versions>10.2

### DISCUSSION THREAD

**Discussion by archithrajan** [FeTS-AI/Front-End#84] [Q&A]
Dear Experts,

Wanted to enquire if a later version of the installer would be compatible with cuda versions > 10.2.

Regards,
Archith

**Comment by sarthakpati** [comment]
Hi,

Thank you for your interest in the FeTS project! 

This project is no longer being actively developed or maintained. Please refer to our Docker builds: https://github.com/orgs/FeTS-AI/packages?repo_name=Front-End 

Cheers,
Sarthak

**Reply by archithrajan** [reply]
Thank you so much for the update.

Regards,
Archith
